pkg/vcdsdk/gateway.go

```diff
@@ -1468,11 +1468,15 @@ func (gm *GatewayManager) IsNSXTBackedGateway() bool {
 // in case the code is unable to determine the required info, it will return false with an error.
 func (gm *GatewayManager) IsUsingIpSpaces() (bool, error) {
 	edgeGatewayName := gm.GatewayRef.Name
-	vdc := gm.Client.VDC
-	if vdc == nil {
-		return false, fmt.Errorf("unable to determine if gateway [%s] is using Ip Spaces or not. nil VDC object in client", edgeGatewayName)
+	edgeGatewayID := gm.GatewayRef.Id
+	clusterOrg, err := gm.Client.VCDClient.GetOrgByName(gm.Client.ClusterOrgName)
+	if err != nil {
+		return false, fmt.Errorf("error retrieving org [%s]: [%v]", gm.Client.ClusterOrgName, err)
+	}
+	if clusterOrg == nil || clusterOrg.Org == nil {
+		return false, fmt.Errorf("unable to determine if gateway [%s] is using Ip Spaces or not; obtained nil org", edgeGatewayName)
```

Comment:
Can we log the name of the org here. Though the org is generally known from the RDE, it has some features such as case sensitivity etc. So it will help to know the exact string of the name of the org that we used to search.

Reply:
added